cmd/gpud/command/command.go

```diff
@@ -111,7 +112,38 @@ nohup sudo gpud run &>> <your log file path> &
 				cli.StringFlag{
 					Name:  "endpoint",
 					Usage: "endpoint for checking in",
-					Value: "mothership-machine-mothership-machine-dev.cloud.lepton.ai",
+					Value: "mothership-machine.app.lepton.ai",
+				},
+			},
+		},
+		{
+			Name:   "kubeconfig",
+			Usage:  "Writes the kubeconfig with gpud.",
+			Action: cmdKubeConfig,
+			Flags: []cli.Flag{
+				cli.StringFlag{
+					Name:  "file",
+					Usage: "file path to output the kubelet config",
+				},
+				cli.StringFlag{
+					Name:  "region",
+					Usage: "region of target cluster",
+				},
+				cli.StringFlag{
+					Name:  "cluster",
+					Usage: "name of target cluster",
+				},
+				cli.StringFlag{
+					Name:  "role",
+					Usage: "role arn",
```

Comment:
```suggestion
					Usage: "role",
```

in case we support non-AWS roles?